docker/bitcoin-regtest/blockchaincfg.json

```diff
@@ -16,5 +16,6 @@
     "slip44": 1,
     "mempool_workers": 8,
     "mempool_sub_workers": 2,
-    "block_addresses_to_keep": 300
+    "block_addresses_to_keep": 300,
+    "mempoolfullrbf": 1
```

Comment:
This is absolutely blind shot at this point. I am still figuring our where to put this config...